docs/source/conf.py

```diff
@@ -80,11 +80,12 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-# TODO: change to [:2] at v1.0
-version = 'master '
-# The full version, including alpha/beta/rc tags.
-# TODO: verify this works as expected
+VERSION = os.environ.get('VERSION', False)
```

Comment:
instead of False here could we just do the following:
`version = os.getenv('VERSION', 'master)`

Reply:
adopted